build/cdc_data/run.sh

```diff
@@ -32,6 +32,16 @@ if [[ $OUTPUT_DIR == "" ]]; then
     exit 1
 fi
 
+if [[ $DATA_RUN_MODE != "" ]]; then
```

Comment:
Oh sorry forgot to explain, I called it `DATA_RUN_MODE` since typically the same env.list file is used for both data and services containers, so if we ever want to add mode for services it could be confusing.

Reply:
Ah - good point! Makes sense.